src/app/anon/Growth Hacks for mobile

```diff
@@ -0,0 +1,52 @@
+Growth Hacks for mobile
+
+Driving people to site and get them to do somthing. Value - prop.
+
+How to Start - Awareness.
+
+Most startups are uninvested in metrics.
+
+Barter System - Y combinator.
+
+Key partner- symbiotic relationship.
+
+Traction - Find somebody who has customers.
+
+Bulding a user base. - Trade advertising, customers. 
+
+Where are the leverage points. 
+
+Who are the early adoptors- Gamers and VR enthusiast.
+
+Optimazaton - Explain your app
+
+Social Media- Pay for a customers who is new. Not existing.
+
+Game Developers Launch games in canada first. To test out the Game.
+
+Walmart deal - open in there stores, open stores in 22 states. to much overhead.
+
+Dont focus on getting featured to quickly.
+
+mobileaction.co rating how visible you really are?
+
+Branch Metrics - which sources are sending how much traffic over and how many have opened the 
+
+What influences download price description.
+
+Be intentional about when you show people promt rating system
+
+Dont just be happy that you got people to pay.. WHats there behavior? Why did they stay?
+
+Talk to happy customers.
+
+Pirate Metrics (ARRRR) - A = Aquistion A = Activity  Most Important R = Retention 
+R = Refferal  R = Rev
+
+Dont agressivliy aquire users if you cant retain them. Dont  stall your launch because it is not perfect.
+
+Frictionless onbording process.
+
+Build test learn.
+
+
```

Comment:
Cool Man... Thanks for the notes... Im going to work on these this weekend
and try to push before monday.

On Fri, Aug 21, 2015 at 1:10 AM, Matt Kinney <kinney.matthewl@gmail.com>
wrote:

> haha.. notes from fast start.. i was coding at the same time.
>
>
> On Thu, Aug 20, 2015 at 8:10 PM, kparks29 <notifications@github.com>
> wrote:
>
>> In src/app/anon/Growth Hacks for mobile
>> <https://github.com/kparks29/stocktracker/pull/3#discussion_r37603301>:
>>
>> > +Be intentional about when you show people promt rating system
>> > +
>> > +Dont just be happy that you got people to pay.. WHats there behavior? Why did they stay?
>> > +
>> > +Talk to happy customers.
>> > +
>> > +Pirate Metrics (ARRRR) - A = Aquistion A = Activity  Most Important R = Retention
>> > +R = Refferal  R = Rev
>> > +
>> > +Dont agressivliy aquire users if you cant retain them. Dont  stall your launch because it is not perfect.
>> > +
>> > +Frictionless onbording process.
>> > +
>> > +Build test learn.
>> > +
>> > +
>>
>> what is this page?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kparks29/stocktracker/pull/3/files#r37603301>.
>>
>
>
>
> --
> Matt Kinney D-)
> www.orangecountyvr.com
> www.socalvr.com
>
>


-- 
Matt Kinney D-)
www.orangecountyvr.com
www.socalvr.com